src/styles/card.css

```diff
@@ -0,0 +1,62 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+  display: block;
+  object-fit: cover;
+  object-position: center;
+  left: 0;
+  right: 0;
+}
+
+.card__name {
+  width: 166px;
```

Comment:
and what if card's width increases?

Reply:
don't know how to fix it. will put on q&a

Reply:
delete width for all elements
You should specify width value only for the card